### ISSUE TITLE
Allure's Issues tag not supported.

### ISSUE THREAD

**Issue by SuitGuy** [Frameworkium/frameworkium-core#3]
More of an improvement.

currently the 

``` java
@Issue("jira ID")
```

works correctly and very well. but occasionally i would like to mark off multiple jira tests with one test.I have some tests that require 2 parts of the system to work together in order to work, if the test is successful i would like it to mark off the results for both of the tests in jira. Allure has the functionality for multiple issue IDs in the form.

``` java
@Issues({
    @Issue("jiraId 1"),
    @Issue("jiraid 2")
})
```

when you then perform the getAnnotation(Issues.class).value() this will return an array of Issue objects that you can iterate over to update Jira. from what i can see, a small edit to the ZapiListener class would make this possible.

**Comment by SuitGuy** [comment]
I have managed to hot fix this functionality into the code myself. but it has not been thoroughly tested with the rest of the framework that uses the Issue tag.

**Comment by jleeh** [comment]
Bumping an old issue, but I'm just resolving any open issues which have been fixed.

Is this referring directly to the zapi integration? Do you know if your code has been included since this comment? If not, what changes did you make?

**Comment by SuitGuy** [comment]
This is referring to the zapi integration. 

from what i can see the logic that updates the jira executions is still only working for 1 issue tag. the changes i made involved changing the getissuesortestcaseid(); method to return an array of the issue tags rather than just the single one. the execution and updater would then execute the update for all issue tags in the array. 

you also need to make changes to ensure that all methods using the @issue tag are still pulled through correctly by the @issues. 

My fix was live on the project i left and if you are able to contact Civ S or Pete R. they should be able to point you to it.

**Comment by civsiv** [comment]
The fix to this was not integrated into Core, but remained on our project in Cocktails. However this is next on my list of things to implement. Waiting on Step Execution to be reviewed and merged before doing this as to not cause conflicts.

**Comment by ham1** [comment]
Is the fix still around?

We will shortly be upgrading to Allure2 but this is just a change of package for the `@Issue` and `@TestCaseId` -> `@TmsLink`

**Comment by ham1** [comment]
Please re-open if this is useful and not fixed with Allure 2 in Frameworkium 3.0 which should be released in the next few weeks.
